### ISSUE TITLE
Under certain zoom levels, context menu doesn't appear when clicking on elements

### ISSUE THREAD

**Issue by var-const** [bpmn-io/bpmn-js#596]
Reproduction steps:
1. Open demo.bpmn.io/new
2. Add a Task (works with other elements, too).
3. Zoom out level by level and try clicking the element after each zoom change.

Result:
1. Context pad doesn't appear under certain zoom levels.
2. Selection looks wrong (the rectangle misses edges) under certain zoom levels (see screenshot).

![6](https://cloud.githubusercontent.com/assets/8216808/17004919/66c37bec-4ea5-11e6-98f7-57e3bb482a1a.png)

**Comment by ricardomatias** [comment]
Closed in favor of bpmn-io/diagram-js#163

**Comment by var-const** [comment]
Note #2 of result ("Selection looks wrong (the rectangle misses edges)") - this looks like a glitch and is not referenced in bpmn-io/diagram-js#163. If you prefer, I can submit a new issue with just this problem.

**Comment by nikku** [comment]
Just to clarify: This is not a bug but a feature.

It does not make sense to show the context-pad beyond certain zoom levels because it will expand / shrink with the element. We _may_ adress this via bpmn-io/diagram-js#163.

**Comment by rmondejar** [comment]
I agree with the explanation, however something like a "focus tool" will be very useful. 

Let me elaborate that : after several discussion with our users, they ask for a tool like this since is a little bit annoying to change the type and move the element with precision, particularly in complex diagrams. In addition, multiple selection (ex. pressing shift) with this tool will allow fine-grained selection.

**Comment by nikku** [comment]
What do you refer to as a `focus tool`?

**Comment by rmondejar** [comment]
I named "focus tool" because I was thinking in a new way to select one item or item selection. However may be "move tool" is a better name, since the idea is change the modeller behaviour to be focused on this selection and allow the user inputs to move o change only this with more precision.

**Comment by nikku** [comment]
I don't understand your explanation. 

* What is the exact problem your users have?
* What is the proposed solution ("focus tool") and how exactly does it solve the problem?

**Comment by rmondejar** [comment]
With this "unnamed tool" the users want to :
* Make easier working with big diagrams
* If one item is selected, then an auto zoom (or maybe with double click) will enable the context menu 
* Move the selection instead of whole diagram by using the cursor keys

**Comment by nikku** [comment]
We've changed the default behavior of overlays (and thus the context pad with bpmn-io/diagram-js#229).

The context pad will now always be visible. It will still scale, making it hardly visible on some zoom levels though. This could be worth another feature request. :wink:

**Comment by nikku** [comment]
Adressed via bpmn-io/diagram-js#229.

**Comment by nikku** [comment]
Added https://github.com/bpmn-io/bpmn-js/issues/719 (min scale for context pad) as a possible improvement
